File: src/main/java/upbrella/be/rent/dto/request/ReturnUmbrellaByUserRequest.java
package upbrella.be.rent.dto.request;

import lombok.*;

import javax.validation.constraints.Size;

@Getter
@Builder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
public class ReturnUmbrellaByUserRequest {

    private long returnStoreId;
    private String bank;
    private String accountNumber;

    @Size(max = 400, message = "최대 400자여야 합니다.")
    private String improvementReportContent;
}

File: src/main/kotlin/upbrella/be/rent/dto/request/ReturnUmbrellaByUserRequest.kt
package upbrella.be.rent.dto.request

import javax.validation.constraints.Size

data class ReturnUmbrellaByUserRequest(
    val returnStoreId: Long = 0,
    val bank: String? = null,
    val accountNumber: String? = null,

    @field:Size(max = 400, message = "최대 400자여야 합니다.")
    val improvementReportContent: String? = null
)